src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -1009,8 +1011,10 @@ bool PerformInteractionRequest::CheckChoiceIDFromRequest(
     const smart_objects::SmartObject& choice_set_id_list) const {
   LOG4CXX_AUTO_TRACE(logger_);
 
+  // cppcheck-suppress variableScope
```

Comment:
FIx variable scope

Reply:
Fixed in 86dbcbdcf96ad8b7cf805c4ca90b57b1a347178b
Please check